fivejars/openy_gated_content
# Shared content module improvements

added fields field_gc_share, field_gc_origin, field_ls_share, field_ls_origin to gc_video, vy_blog_post, live_stream, virtual_meeting bundles
mass action "Share to Virtual Y", "Unshare from Virtual Y" for node, eventseries and eventinstance entity types

steps to check:
- enable openy_gc_shared_content module
- go to /admin/content
- go to /admin/content/events/series
- go to /admin/content/events/instances
- check Share to Virtual Y, Unshare from Virtual Y mass actions
